components/03-modules/brief-info/BriefInfo.stories.js

```diff
@@ -0,0 +1,22 @@
+import { storiesOf } from '@storybook/vue'
+
+import App from '../../01-globals/app/App.vue'
+import AlpacaBriefInfo from './BriefInfo.vue'
+
+import briefInfo from './mocks/brief-info.json'
+
+storiesOf('Modules/Brief Info', module).add('Default', () => ({
+  components: { App, AlpacaBriefInfo },
+  data: () => ({
+    briefInfo
```

Comment:
It will be great to rename this component, this name is ultra generic.

Reply:
Hmm... do you have any idea for the new name? :D

Reply:
🤷‍♂️

Reply:
Let's put it on some TODO list, we will back to this later.